Uplift/Controllers/HomeViewController.swift

```diff
@@ -62,7 +62,8 @@ class HomeViewController: UIViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        navigationController?.tabBarController?.tabBar.isHidden = false
+//        navigationController?.tabBarController?.tabBar.isHidden = true
```

Comment:
Just double checking, this is a temporary change right?

Reply:
yup!